src/backend/utils/auto_launch.ts

```diff
@@ -0,0 +1,35 @@
+import AutoLaunch from 'auto-launch'
+import { GlobalConfig } from 'backend/config'
+import { ipcMain } from 'electron'
+
+export const hyperPlayAutoLauncher = new AutoLaunch({
+  name: 'HyperPlay',
+  path: process.execPath
+})
+
+export async function updateAutoLaunch(enable: boolean) {

```

Comment:
nitpick: can we make the argument named to make it easier to see we're enabling or disabling it?

```ts
function  updateAutoLaunch(mode: 'enabled' | 'disabled') {
  if (mode === 'enabled')  { ... } 
  else { ... }
}

updateAutoLaunch('enabled');
updateAutoLaucnh('disabled');
```

Reply:
I think boolean makes more sense here since it is a toggle option with only two states 

I also would have to change the param, the call, the settings type, etc. for this. Not sure if it's worth it